packages/node_modules/@webex/plugin-meetings/src/meeting-info/util.js

```diff
@@ -148,6 +148,7 @@ MeetingInfoUtil.getSipUriFromHydraPersonId = (destination, webex) => webex.peopl
   }
   throw new ParameterError('Hydra Id Lookup was an invalid hydra person id.');
 }).catch((err) => {
+  LoggerProxy.logger.error(`MeetingInfo:util#MeetingInfoUtil.getSipUriFromHydraPersonId --> getSipUriFromHydraPersonId ${err} `);
```

Comment:
```suggestion
  LoggerProxy.logger.error(`Meeting-info:util#MeetingInfoUtil.getSipUriFromHydraPersonId --> getSipUriFromHydraPersonId ${err} `);
```

To match @abailey-wwt's changes https://github.com/webex/webex-js-sdk/pull/1710/files#diff-c58ab53994c8db2c15c286b8f77fd9acR54